examples/with-create-react-app/apps/docs/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```

Comment:
Can we have one gitignore at the root?

Reply:
This also falls under my, "stick with the generator" opinion. Nested `.gitignore` is valid, fine, and clear. 🤷‍♂️